src/tools/pkcs15-init.c

```diff
@@ -720,21 +720,42 @@ static const struct alg_spec alg_types_sym[] = {
 	{ NULL, -1, 0 }
 };
 
+// clang-format off
+/* RSA can have a number , default is 2048 */
+/* EC require a curve name */
+/* EDDSA and XEDDSA  without a size require a size or curve name or OID */
+/* other EDDSA and XEDDSA can be used  alone */
 static const struct alg_spec alg_types_asym[] = {
-	{ "rsa",	SC_ALGORITHM_RSA,	1024 },
+	{ "rsa",	SC_ALGORITHM_RSA,	2048 }, /* new default */
```

Comment:
Although this is a big jump, 3000 bits should be the recommended minimum key size nowadays

Reply:
OK default RSA size in 3072.